asv_bench/benchmarks/frame_ctor.py

```diff
@@ -81,3 +81,15 @@ def setup(self, nrows):
     def time_frame_from_records_generator(self, nrows):
         # issue-6700
         self.df = DataFrame.from_records(self.gen, nrows=nrows)
+
+
+class FromNDArray(object):
+
```

Comment:
are there already list-like constructor asv's here?  and from dict?

Reply:
no idea (unrelated to this PR)

Reply:
and the answer is?

Reply:
no idea, as above

Reply:
well since you are adding this one, should be very easy to .parametrize and add similar to the Series case, at least a few simple ones.

Reply:
Sure, some simple ones are present in ``frame_ctor.py``, above the code I moved from ``ctor.py``. I can open an issue to remind adding more.

Reply:
yes

Reply:
#18840